draft-iab-protocol-maintenance.md

```diff
@@ -440,6 +476,20 @@ For instance, the INADEQUATE_SECURITY error code and associated requirements in
 HTTP/2 {{H2}} resulted in improvements in the security of the
 deployed base.
 
+When the protocol involves shared state between multiple parties (often refered
+to as a connection), the fatal errors described in this section are best sent
+over the network as explicit error messages. Additionally, those error messages
```

Comment:
I don't think that this depends on shared protocol state.  ICMP defines exactly the sort of feedback you describe here, where the offending party is informed of what they did that was "wrong".

```suggestion
A notification for a fatal error is best as explicit error messages to the entity that
made the error. Error messages
```